hip-service/Link/Patient/PatientVerification.cs

```diff
@@ -1,27 +1,84 @@
-#nullable enable
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
 using System.Threading.Tasks;
 using hip_service.OTP;
 using HipLibrary.Patient.Model.Response;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Optional;
 
 namespace hip_service.Link.Patient
 {
     public class PatientVerification: IPatientVerification
```

Comment:
Do we have test for this class as HTTPClient is being instantiated with in the method itself?

Reply:
No, Searching google for a tutorial on the same.

Reply:
Added testcases